docs/statistics/highest-false-positives-last-month/index.html

```diff
@@ -343,13 +343,13 @@ <h1 class="text--white">Highest False Positives Last Month</h1>
           13
         </td>
         <td >
-          <a href="/docs/alerts/10055/">CSP</a>
+          <a href="/docs/alerts/40018/">SQL Injection</a>
         </td>
         <td >
           release
         </td>
         <td >
-          Passive
+          alert
```

Comment:
Should be `Active`?

Reply:
Ah, its a bug in my script for updating the alerts table ;)

Reply:
FYI my alert table gen script is local - I'll have to PR that at some point..